plugins/portals-list.js

```diff
@@ -134,6 +154,43 @@ window.plugin.portalslist.fields = [
     },
     defaultOrder: -1,
   },
+  { 
+    title: "V/C",
+    value: function(portal) { return portal.options.guid; }, // we store the guid, but implement a custom comparator so the list does sort properly without closing and reopening the dialog
+    sort: function(guidA, guidB) {
+      return window.plugin.portalslist.visitedValue(guidA) - window.plugin.portalslist.visitedValue(guidB);
+    },
+    format: function(cell, portal, guid) {
+      var info = window.portals[guid].options.data.history;
+      if(!info) info = { visited: false, captured: false, scoutControlled: false};
+
+      $(cell).addClass("portal-list-history alignC");
+      if (info.captured){
+        cell.append (String.fromCodePoint(0x1F534)); // red dot
+      } else {
+        if (info.visited) {
+          cell.append (String.fromCodePoint(0x1F7E1)); //yellow dot
+        } else {
+          cell.append (String.fromCodePoint(0x26AA)); // white dot
+        }
+      }
+
+    }
```

Comment:
```suggestion
    title: "V/C",
    value: function(portal) { window.plugin.portalslist.visitedValue(portal.options.guid); },
    format: function(cell, portal, value) {
      $(cell).addClass("portal-list-history alignC");
      var codePoint = [0x26AA, 0x1F7E1, 0, 0x1F534][value];
      cell.append (String.fromCodePoint(codePoint));
    }
```

Reply:
the `[0x26AA, 0x1F7E1, 0, 0x1F534]` I suggest will benefit to be a variable in the plugin scope (`portalList.VCSCodepoints` or a definitely better name)